### PR TITLE
[15.0][ADD] mrp_2_steps_consumed_qty_sync: New module

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa  TT50868

ping   @carlosdauden @chienandalu

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: mrp_2_steps_consumed_qty_sync/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module allows update the quantities consumed from a production order components
```
Comment (chienandalu) [inline mrp_2_steps_consumed_qty_sync/readme/DESCRIPTION.rst]:
```suggestion
This module allows to update the quantities consumed from a production order components
```

**Review by chienandalu — COMMENTED** [review]
I can't make it work in runboat. Steps I follow:

- I set 2 steps production
- Then I make a MO for 5 [FURN_8855] Drawer and confirm
- The pick picking is created, so I go to it and update the done quantities for the components: i raise them from 5 to 8
- I validate the pick picking
- :point_right: But it doesn't show up in the MO lines :man_shrugging:

**Comment by sergio-teruel** [comment]
@chienandalu Fixed

**Review by chienandalu — APPROVED** [review]
Tested succesfully now :+1:

**Comment by chienandalu** [comment]
Should this behavior be proposed to Odoo master?

**Comment by sergio-teruel** [comment]
May be!! I'm not sure!!

**Comment by chienandalu** [comment]
I mean, it's indeed really weird that those pick picking quantities have no effect in the MO

**Pull request merged**